modules/openapi-generator/src/main/resources/php/api.mustache

```diff
@@ -472,16 +472,24 @@ use {{invokerPackage}}\ObjectSerializer;
         $multipart = false;
 
         {{#queryParams}}
+
         // query params
-        {{#collectionFormat}}
+        {{#isExplode}}
+        if (${{paramName}} !== null) {
+            $queryParams['{{baseName}}'] = ${{paramName}};
+        }
+        {{/isExplode}}
+        {{^isExplode}}
         if (is_array(${{paramName}})) {
-            ${{paramName}} = ObjectSerializer::serializeCollection(${{paramName}}, '{{collectionFormat}}', true);
+            ${{paramName}} = ObjectSerializer::serializeCollection(${{paramName}}, '{{#style}}{{style}}{{/style}}{{^style}}{{#collectionFormat}}{{collectionFormat}}{{/collectionFormat}}{{/style}}', true);
```

Comment:
Unfortunately, this code is broken due to incomplete implementation of the style property, in PR #4042; style will always be empty, since this code is looping queryParams, and queryParams gets copies of the parameters in allParams. Since the style property is not included in the copy() method for the CodegenParameter class, its value will always be empty... :/